### PR TITLE
Adds a new macaronic language

### PR DESCRIPTION
**Pull request by LordFowl**
Implements Encoded Nral'sinta'maas-songnet Basic-pidgin, a macaronic language that draws from all languages, known and unknown, past, present, and future, that aroused from the void in 2450, implanting itself like a parasite in the minds of Tajara stowaways so that it could reproduce through memetic propagation throughout the entirety of the known galaxy, composing itself as a super-lifeform with intellect and power superseding all of that known to man.

image

 -  [x] Lore review

### CODE REVIEW

**Comment by ParadoxSpace** [comment]
speedmerge tbh

**Comment by chaoko99** [comment]
image

**Comment by LordFowl** [comment]
thumps up from a lore deputy - that's lore review done.

**Comment by skull132** [comment]
Lore has reviewed it as a no-go and I'm not seeing the point of this PR.

File: code/modules/mob/language/station.dm [hunk 1]
```diff
@@ -281,6 +281,33 @@
...
+			"den", "deng", "di", "dian", "diao", "die", "ding", "diu", "dong", "dou", "du", "duan", "dui", "dun", "duo", "e", "shai", "shan", "shang", "shao", "she", "shei", "shen", "sheng", "shi", "shou", "shu", "shua", "shuai", "shuan", "shuang", "shui",
+			"ve", "wa", "all", "and", "are", "but", "ent", "era", "ere", "eve", "for", "had", "hat", "hen", "her", "hin", "blah","blah","blah","bleh","meh","neh","nah","wah",
+			"lorem", "ipsum", "dolor", "sit", "amet", "consectetur", "adipiscing", "elit", "in", "culpa", "qui", "officia", "deserunt", "mollit", "anim", "id", "est", "laborum",
+			"slo","nik","ko","zels","het","zlo","nis","iv","da","ati","yib","ban","dup","sha","ansh","nou","nec","zby", "ci", "m","n","gh","h","l","s","r","a","e","i","o","u",
+			"ti","ti","ti","hi","hi","ki","ki","ki","ki","ya","ta","ha","ka","ya","chi","cha","kah", "SKRE","AHK","EHK","RAWK","KRA","AAA","EEE","KI","II","KRI","KA", "ire","ego","nahlizet","certum","veri","jatkaa","mgar","balaq", "karazet", "geeri",,
```
Comment (Karolis2011) [inline code/modules/mob/language/station.dm]: Two commas, I disapprove of that.

**Pull request closed without merging**